deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright 2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+VENDOR=oracle
+DRIVER=oci
+
+driver_dir=$VENDOR${VENDOR:+"~"}${DRIVER}
+if [ ! -d "/flexmnt/$driver_dir" ]; then
+  mkdir "/flexmnt/$driver_dir"
+fi
+
+cp "/$DRIVER" "/flexmnt/$driver_dir/.$DRIVER"
+mv -f "/flexmnt/$driver_dir/.$DRIVER" "/flexmnt/$driver_dir/$DRIVER"
+
+while : ; do
+  sleep 3600
```

Comment:
Is sleeping forever ok? I've seen this sort of thing exit and have never restart set?

Reply:
Yes, it's ok and the preferred method for this type of action (cni installers do it as well) and what upstream recommends for the flex volume plugin.

https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/flexvolume-deployment.md#driver-deployment-script

I would add though you need to make sure this gracefully can be shutdown. `sleep 3600` won't respect sigterm's. If you do 

```bash
sleep 3600 &
wait
```
it will.